whois-api/src/main/java/net/ripe/db/whois/api/fulltextsearch/ElasticFulltextSearch.java

```diff
@@ -136,7 +138,16 @@ protected SearchResponse doSearch() throws IOException {
     }
 
     private org.elasticsearch.action.search.SearchResponse performFulltextSearch(final SearchRequest searchRequest) throws IOException {
-        return elasticIndexService.getClient().search(getFulltextRequest(searchRequest), RequestOptions.DEFAULT);
+        org.elasticsearch.action.search.SearchResponse response;
+        try {
+            response = elasticIndexService.getClient().search(getFulltextRequest(searchRequest), RequestOptions.DEFAULT);
```

Comment:
You can still return from inside the try block, no need for a `response` variable (it's never used)